packages/odyssey-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -114,8 +115,9 @@ export const Checkbox = withTheme(
               {indeterminate ? <SubtractIcon /> : <CheckIcon />}
             </span>
           </span>
-
-          {label}
+          <span
+            dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(label) }}
+          />
```

Comment:
We shouldn't use this I don't think. It will most-likely cause issues with security and nonce stuff. You won't find out until someone consumes Odyssey, and the Monolith runs its content-security checks.